Project/MyGameLibrary/Story/Option.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace MyGameLibrary.Story
+{
+    public class Option
+    {
+        public string OptionText { get; set; }
+        public string OptionBackendMarkup { get; set; }
+        private float _OptionDarkenBrightenPercent = (float).20; //Must be between 0-1
+        private bool _OptionFocused;
+        public bool OptionFocused
+        {
+            get
+            {
+                return _OptionFocused;
+            }
+
+            set
+            {
+                if(!_OptionFocused && value) //If set value is true and was previously false
+                {
+                    #region Darken BackColor
+                    float correction  = 1 - _OptionDarkenBrightenPercent;
+                    int a = OptionLabel.BackColor.A;
```

Comment:
could you move this up so that you don't have to have it in both of the if statements?